cmd/configui/index.html

```diff
@@ -240,14 +244,14 @@ <h5 class="modal-title" id="exampleModalLabel">Add Splunk HEC </h5>
                 // Uploaded file is invalid if either of the 2 conditions mentioned
                 // above (3 values in each line and  first value should be a valid IP)
                 // are not satisfied
-                if (isValidIP==false || isValidfieldsFlag==false){
-                    isValidCSVFlag = false
-                }else{
+                if (isValidIPFlag && isValidfieldsFlag){

```

Comment:
can be simplified here as well. 
isValidCSVFlag = isValidIPFlag && isValidfieldsFlag